src/tls/Makefile.common

```diff
@@ -1,10 +1,33 @@
-HACL_HOME?=../../../hacl-star
-FSTAR_HOME?=../../../FStar
-MITLS_HOME?=../..
-KREMLIN_HOME	?= ../../../kremlin
-MLCRYPTO_HOME	?= ../../../MLCrypto
+HACL_HOME       ?= ../../../hacl-star
+FSTAR_HOME      ?= ../../../FStar
+MITLS_HOME      ?= ../..
+KREMLIN_HOME    ?= ../../../kremlin
+MLCRYPTO_HOME   ?= ../../../MLCrypto
+FLAVOR          ?= Model
+CONCRETE_FLAVOR ?=
 
-INCLUDE_PATHS= \
+SHELL=/bin/sh
+
+# Project-wide Prerequisites
+
+ifeq ($(OS),Windows_NT)
+  LIB_EXT?=.dll
+else
+  LIB_EXT?=.so
+endif
+
+# PREREQUISITES= \
```

Comment:
can we remove commented out lines?

recursive makes are generally prohibited, since it defeats the purpose of the everest script, and may result in disjoint make instances running the same job in parallel, which is a recipe for disaster

Reply:
Sure, do you have a better idea for project-wide prerequisites?